code/modules/clothing/shoes/wheelys.dm

```diff
@@ -26,7 +26,7 @@
 	if(!isliving(user))
 		return
 	if(!istype(user.get_item_by_slot(ITEM_SLOT_FEET), /obj/item/clothing/shoes/wheelys))
-		to_chat(user, span_warning("You must be wearing the wheely-heels to use them!"))
+		to_chat(user, span_warning("You must be wearing \the [src] to use [src.p_them()]!"))
```

Comment:
```suggestion
		to_chat(user, span_warning("You must be wearing [src] to use [p_them()]!"))
```

Reply:
Could also be a balloon alert if you wanted

Reply:
Sounds good.